nbextensions/appCell2/widgets/appCellWidget-fsm.js

```diff
@@ -686,95 +666,6 @@ define([], () => {
                     mode: 'processing',
                     stage: 'running',
                 },
-                {
-                    mode: 'processing',
-                    stage: 'partial-complete',
-                },
-
-                {
-                    mode: 'canceled',
-                },
-                {
-                    mode: 'canceling',
-                },
-                {
-                    mode: 'success',
-                },
-                {
-                    mode: 'error',
-                    stage: 'runtime',
-                },
-                {
-                    mode: 'editing',
-                    params: 'complete',
-                    code: 'built',
-                },
-                {
-                    mode: 'internal-error',
-                },
-            ],
-        },
-        // processing - partial complete
```

Comment:
this is actually the removal of the `processing - partial complete` state, but the diff algorithm is greedy